var/spack/repos/builtin/packages/powerapi/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Powerapi(AutotoolsPackage):
+    """This software is a reference implementation of the PowerAPI"""
+
+    homepage = "https://powerapi.sandia.gov/"
+    git      = "https://github.com/pwrapi/pwrapi-ref.git"
+
+    version('1.1.1', commit='93f66dfa29f014067823f2b790a1862e5841a11c')
+
+    variant('hwloc', default=False, description='Build hwloc support')
+    variant('debug', default=False, description='Enable debug support')
+    variant('mpi',   default=False, description='Enable MPI support')
+    variant('package', default=False, description='Enable package support')
+    variant('gnu-ld', default=False, description='Assume GNU compiled uses gnu-ld')
```

Comment:
Huh?  What is this for?  And how does it interact with other package / etc?

Reply:
I removed the `package` variant. Including it was a mistake on my part. I am not a maintainer of powerapi, but our environment does include it so I was trying to add the package to spack and at the time I misread the list of configuration flags and I thought it was a config option.